corehq/apps/app_manager/models.py

```diff
@@ -986,9 +986,71 @@ def validate(self, value, required=True):
 
 
 class NavMenuItemMediaMixin(DocumentSchema):
+    """
+        Language-specific icon and audio.
+        Properties are map of lang-code to filepath
+
+        Convention (for old docs): Unless a specific language key exists,
+        by default, media for that language is same as that for English
```

Comment:
Similarly, I don't quite follow the places where this mixin uses 'en' - what happens if the app doesn't use english?